spec/models/user_spec.rb

```diff
@@ -0,0 +1,33 @@
+require '../rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'austin', bio: 'student', post_counter: 1) }
```

Comment:
- [ ] An error is thrown when running your tests because post_counter` does not exist. It is `posts_counter`.
![image](https://github.com/Obimbo07/tdn/assets/45460192/750616c4-ff06-4392-8aca-55f722e3a5a9)
Please fix this in all other tests in your spec/models folder.